roxctl/netpol/generate/generate_test.go

```diff
@@ -96,6 +96,30 @@ func (d *generateNetpolTestSuite) TestGenerateNetpol() {
 			expectedWarnings:        []string{},
 			expectedErrors:          []string{},
 		},
+		"should report bad port name": {
+			inputFolderPath: "testdata/minimal",
+			dnsPort:         "bad@chars",
+
+			expectedValidationError: errox.InvalidArgs,
+			expectedWarnings:        []string{},
+			expectedErrors:          []string{},
+		},
+		"should report bad port number": {
+			inputFolderPath: "testdata/minimal",
+			dnsPort:         "100000",
+
+			expectedValidationError: errox.InvalidArgs,
+			expectedWarnings:        []string{},
+			expectedErrors:          []string{},
+		},
+		"should report 0 as a bad port number": {
+			inputFolderPath: "testdata/minimal",
+			dnsPort:         "0",
+
+			expectedValidationError: errox.InvalidArgs,
+			expectedWarnings:        []string{},
+			expectedErrors:          []string{},
+		},
```

Comment:
Could we also add a case with `dnsPort: ""`?

Reply:
and another one with a negative integer value :)

Reply:
Added both